frontend/src/components/pages/Account.js

```diff
@@ -28,6 +30,11 @@ export function Account() {
             .then((data) => setUserInfo(data.user))
             .catch((error) => setError(error.message));
     }, []);
+    const handleLogout = () => {
```

Comment:
Here, we also want to remove the userId from the cookie. Also, a way to reload the way is to add window load after the navigate().

Example:
```js
const handleLogout = () => {
        Cookies.remove('token');
        Cookies.remove('userId');
        setUserInfo(null)
        navigate('/login', { replace: true });
        window.location.reload();
    };
```